src/playbooks/template-language.md

```diff
@@ -0,0 +1,125 @@
+| `outputs` | A merged view of all the "outputs" from all previous steps. Outputs from later steps override outputs from earlier steps. |
+
+### The `trigger` object
+
+The `trigger` variable is a "Step Results" object and has the following properties:
+
+| Variable | Description |
+| -------- | ----------- |
+| `trigger.id` | The ID of the trigger |
+| `trigger.outcome` | The outcome of the trigger. Will always be `Succeeded`. |
+| `trigger.outputs` | A table of output values from the trigger. The values in this table vary depending on the trigger. |
+
+### The `previous` object
+
+The `previous` variable is also "Step Results" object and has the following properties:
+
+| Variable | Description |
+| -------- | ----------- |
+| `previous.id` | The ID of the previous step in the playbook |
+| `previous.outcome` | The outcome of the previous step in the playbook. Will always be `Succeeded`. |
+| `previous.outputs` | A table of output values from the previous step in the playbook. The values in this table vary depending on the step. |
+
+> [!NOTE]
+> For the first step in a playbook, `previous` will be identical to the `trigger`.
+
+### The `outputs` object
+
+The `outputs` variable is a table of all the outputs from all previous steps in the playbook.
+Unlike, say, `trigger.outputs` which is **just** the outputs from the trigger, or `previous.outputs` which is **just** the outputs from the previous step, this is a merged view of all `outputs` from all steps that have executed.
+If a later step has an output with the same name as an earlier step, the later step's output will override the earlier step's output.
+
+## Some examples
+
+Here are some examples of triggers and steps and how you can use their outputs in templates.
+This isn't a full reference of all the outputs available, but it gives a brief overview of what to expect.
+
+### Accessing the channel that triggered the playbook
+
+If you use a trigger like "Conversation Started", "Reaction Added", "Abbot Added to Channel", or other triggers that are caused by events related to Slack messages, the Slack channel in which that event occurred is available in the `trigger.outputs.channel` object.
+That object has the following properties:
+
+| Variable | Description |
+| -------- | ----------- |
+| `trigger.outputs.channel.id` | The ID of the channel |
+| `trigger.outputs.channel.name` | The name of the channel |
+
+If you want to mention the Channel in a Slack message, use Slack's [linking syntax](https://api.slack.com/reference/surfaces/formatting#linking-channels):
+
+```
+Hey everyone, we got a new customer. They're in <#{{ trigger.outputs.channel.id }}>!
+```
+
+### Accessing the Customer that triggered the playbook
+
+If you trigger a playbook using the "New Customer Created" trigger, a `customer` output is available that contains information about the customer:
+
+| Variable | Description |
+| -------- | ----------- |
+| `trigger.outputs.customer.id` | The ID of the customer |
```

Comment:
Out of curiosity, is there ever a reason that an end user would need to know the ID of a customer?

Reply:
(I'm not saying it shouldn't be in the docs, just curious about its usefulness to end users)

Reply:
Not really, no

Reply:
I'll remove it from the doc, it's really not useful information.